### DISCUSSION TITLE
Connection with low-latency(OME) server failed - after 0.16.0

### DISCUSSION THREAD

**Discussion by githubetic** [OvenMediaLabs/OvenMediaEngine#1637] [Help] [answered]
Hello,

It´s my first post in Oven Media Engine and I would like to start by thanking all the people who have made this wonderful and scalable streaming platform possible.

This time I just wanted to know if it is necessary to do some special configuration on the OvenMediaEngine or OvenPlayer side to try to avoid the ‘**Connection with low-latency(OME) server failed**’ message.

This only happens to me **from version 0.16.0 onwards**.

**NOTES**: 
- If I click play in OvenPlayer after it show the error message, the webrtc streaming begin instanly.
- I´m using OvenMediaEngine Dockerized and player via WebRTC

**[v0.16.1]**
_[Compatibility]_

Compatible with dependencies(prerequisites.sh) installed in 0.14.9 or later
Compatible with Server.xml used in 0.12.6 or later
Compatible with version 0.15.12 or higher of Origin or Edge

_[Added]_

Added support for multiple hardware acceleration.
Added mp3 codec decoder.

_[Changed]_

Due to performance issues, the logic that holds request sessions until WebRTC and LLHLS streams are ready has been removed.
Changed the NVCC header library installation path.
Deprecated HardwareAccelerator option in Server.xml and changed to HWAccels.

_[Improved]_

Improved ome_docker_launcher.sh to execute without overwriting the configuration file.

_[Fixed]_

Fixed an issue that could cause OVT Publisher to crash intermittently
Fixed an issue that could cause a crash in H.264 when the payload size was 0.
Fixed OriginMapStore deadlock issue.
Fixed issue where device could not be activated in Docker with NVIDIA GPU support.

**Comment by getroot** [comment, marked as answer]
Previously, OME would hold the playback request session until the incoming stream from OVT was ready. This was removed because it caused serious performance issues. Instead, it would be better to add functionality to output a different message and retry if the player gets a 201 CREATED.

**Reply by githubetic** [reply]
The truth is that seeing the changes in version 0.16.1, that was precisely the change that made me most suspicious and that could be affecting the new response of the Player. 

I also have to say that the same version 0.16.1 of OME on a Docker running on a hypervisor based on Proxmox _(underneath a similar HOST)_, does not give me the same problem. The other test is over the Public Cloud of OVH.

_NOTE: I run both tests on Portainer._

Thanks for your help.
